### PR TITLE
fix: client extrinsics operation

### PR DESCRIPTION
**Pull request by thadouk**
## Proposed changes
This commit addresses issues with extrinsic submission in the runtime:

- Registers the extension to the instance submitting the extrinsic.
- Updates signature validation.
- Simplifies interface and error handling via the runtime API, following the report_equivocation style.

### CODE REVIEW

File: pallets/avn/src/lib.rs [hunk 1]
```diff
@@ -393,13 +393,21 @@ impl<T: Config> Pallet<T> {
...
+            warn!("Signature validation failed, account {:?}, is not validator", validator);
             return false
         }
 
         // check signature (this is expensive so we do it last).
         let signature_valid =
             data.using_encoded(|encoded_data| validator.key.verify(&encoded_data, &signature));
 
+        debug!(
+            "🪲 Validating signature: [ data {:?} - account {:?} - signature {:?} ] Result: {}",
```
Comment (nahuseyoum) [inline pallets/avn/src/lib.rs:405]: is the emoji a `bug`? :smile:

File: pallets/eth-bridge/src/lib.rs [hunk 2]
```diff
@@ -894,7 +894,7 @@ pub mod pallet {
                     },
                 Call::submit_ethereum_events { author, events_partition, signature } =>
                     if AVN::<T>::signature_is_valid(
-                        &encode_eth_event_submission_data(
+                        &(
```
Comment (nahuseyoum) [inline pallets/eth-bridge/src/lib.rs:897]: why are we removing the function call? Won't this potentially cause a mistmatch?

**Review by nahuseyoum — APPROVED** [review]

File: pallets/avn/src/lib.rs (hunk 1 above)
Comment (thadouk) [reply pallets/avn/src/lib.rs:405]: cause it is **de**-bug level of info :tada:

File: pallets/eth-bridge/src/lib.rs (hunk 2 above)
Comment (thadouk) [reply pallets/eth-bridge/src/lib.rs:897]: its a missmatch now. Validation expects a tupple, not encoded data.

**Pull request merged**